content/en/getting_started/code_analysis/_index.md

```diff
@@ -302,6 +308,67 @@ Click on a service to access information about CI pipelines from Pipeline Visibi
+
+1. [Identifying the code location associated with a service using the Service Catalog.](#identifying-the-code-location-in-the-service-catalog)
+2. [Detecting usage patterns of files within additional Datadog products.](#detecting-file-usage-patterns)
+3. [Searching for the service name in the file path or repository.](#detecting-service-name-in-paths-and-repository-names)
+
+If one method succeeds, no further mapping attempts are made. Each mapping method is detailed below.
+
+#### Identifying the code location in the Service Catalog
+
+The schema version `v3` and later of the Service Catalog allows you to add the mapping of your code location for your service. The `codeLocations` section specifies the location of the repository containing the code and its associated paths.
+
+The `paths` attribute is a list of [globs][14]
+that should match paths in the repository.
+
+{{< code-block lang="yaml" filename="service.datadog.yaml" collapsible="true" >}}
+apiVersion: v3
+kind: service
+metadata:
+  name: my-service
+datadog:
+  codeLocations:
+    - repositoryURL: https://github.com/myorganization/myrepo.git
+      paths:
+        - /path/to/service/code/**
+{{< /code-block >}}
+
+
+#### Detecting file usage patterns
+
+Datadog detects file usage in additional products such as Error Tracking and associate
+files with the runtime service. For example, if a service called `foo` has
+a log entry or a stack trace containing a file with a path `/modules/foo/bar.py`,
+it associates files `/modules/foo/bar.py` to service `foo`.
+
+#### Detecting service name in paths and repository names
+
+Datadog detects service names in paths and repository names, and associates the file with the service if a match is found.
+
+For a repository match, if there is a service called `myservice` and
+the repository URL is `https://github.com/myorganization/myservice.git`, then,
+it associates `myservice` to all files in the repository.
+
+If no repository match is found, Datadog attempts to find a match in the
+`path` of the file. If there is a service named `myservice`, and the path is `/path/to/myservice/foo.py`, the file is associated with `myservice` because the service name is part of the path. If two services are present
+in the path, the service name the closest to the filename is selected.
+
+
+### Linking teams to code violations and libraries
+
+Datadog automatically associates the team attached to a service when a code violation or library issue is detected. For example, if the file `domains/ecommerce/apps/myservice/foo.py`
+is associated with `myservice`, then the team `myservice` will be associated to any violation
+detected in this file.
+
+If no services or teams are found, Datadog uses the `CODEOWNERS` [file][15]
+in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. 
+
+**Note**: You must accurately map your Git provider teams to your [Datadog teams][16] for this feature to function properly.
+
```

Comment:
Can we put this section somewhere else than getting started since it's related to more advanced use of the tool?

Reply:
@juli1 Would it make sense to add this content with an H3 header like `Link services and teams to code violations` in the [Set up Code Analysis on your repository](https://docs.datadoghq.com/code_analysis/?tab=datadog#set-up-code-analysis-on-your-repository) section? This way, the content is still relevant to the reader and would not exist on this Getting Started page. Thanks!